tslint.json

```diff
@@ -1,5 +1,6 @@
 {
   "extends": ["tslint:latest", "tslint-react"],
   "rules": {
+    "no-implicit-dependencies": [true, "dev"]
```

Comment:
Why was this rule added?

Reply:
Man you beat my comment by a minute! As I mentioned below, this was to import from Enzyme, which still feels like a `devDependency` to me. Not sure if this is the best fix.

Reply:
Enzyme is definitely a devDependency. In theory, we only want to allow imports from devDependencies in test code, whereas this allows it everywhere. If we switched to a model with all the tests under a single test folder then I think we could have a local config that would allow it, but with the tests next to the code, the only other option would be adding individual exceptions in code for every import, which would be worse.